scripts/cutover_to_pg.sh

```
@@ -0,0 +1,135 @@
+  --name TempCutoverAccess \
+  --output none 2>/dev/null
+log "Delta-sync complete. Temporary access removed."
+
+# --- Phase 3: Cutover ---
+log "=== Phase 3: Cutover ==="
+
+log "Backing up current .env on VM1..."
+ssh "$VM1_HOST" "cp $APP_DIR/.env $APP_DIR/.env.sqlite.backup"
+
+log "Updating .env with PostgreSQL credentials..."
+ssh "$VM1_HOST" "cat > $APP_DIR/.env << ENVEOF
```

Shellcheck (0.11.0)

[info] Note that, unescaped, this expands on the client side.

(SC2029)